src/server/journal/types.h

```diff
@@ -46,16 +46,22 @@ struct Entry : public EntryBase {
 };
 
 struct ParsedEntry : public EntryBase {
+  // Represents an owned version of a command.
+  struct OwnedCommand {
+    std::unique_ptr<char[]> buffer;
+    CmdArgVec parts;
+  };
+
```

Comment:
I'm using a `unique_ptr` and not a string because I wan't to make sure it is not copied a single time since the moment of construction, because this would brake the pointers inside the arg vec